.gitignore

```diff
@@ -1,7 +1,8 @@
 # Ant
 /ant-out/
 
-# IntelliJ build
+# IntelliJ
+.idea
```

Comment:
I think it's better to not have this change here. We'd like to keep a clean fork that can merge facebook master easily.

Reply:
Good call. Maybe we try to push this change directly up to Facebook's repo?

Reply:
👍